routes/agreement-1/agreement-1-fr.njk

```diff
@@ -37,13 +49,13 @@
       By participating in this study, you understand that:
     </p>
     <ul>
-      {% if data.compensation == "No" %}
-        <li>You are volunteering to participate. You can stop at any time for any reason, without any consequences.</li>
-      {% else %}
+      {% if data.compensation == "Yes" %}
```

Comment:
The changes in this file are copied over from the en agreement

Reply:
oppsie